cli/tests/test_operations.rs

```diff
@@ -70,7 +72,7 @@ fn test_op_log() {
     "###);
     insta::assert_snapshot!(
         test_env.jj_cmd_failure(&repo_path, &["log", "--at-op", "@---"]), @r###"
-    Error: The "@---" expression resolved to no operations
+    Error: Workspace "default" doesn't have a working copy
```

Comment:
Maybe better to update the query to `@----`. The new error is also interesting, but I think it is the test for empty op

Reply:
Oops, yes (over-aggressive use of `insta test --accept`)